python/vyos/config_mgmt.py

```diff
@@ -0,0 +1,664 @@
+# Copyright 2023 VyOS maintainers and contributors <maintainers@vyos.io>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import re
+import sys
+import pwd
+import gzip
+import logging
+from typing import Optional, Tuple, Union
+from argparse import ArgumentParser, REMAINDER
+from shutil import copy
```

Comment:
I have a feeling that quite a few of these may be better off as local imports inside functions that use them, in general.

Reply:
Several other imports made local, where appropriate; thanks !